basic_gdeploy_doc.md

```diff
@@ -0,0 +1,298 @@
+#### Introduction of gdeploy and how to use gdeploy to configure glusterfs
+-----
+
+This document is intended to give a basic overview of the gdeploy tool like,
+what is gdeploy, what does it do, and how you can use it.
+
+##### What is glusterfs?
+
+GlusterFS is a scale-out network-attached storage file system. It has found applications 
+including cloud computing, streaming media services, and content delivery networks. GlusterFS 
+was developed originally by Gluster, Inc., then by Red Hat, Inc., after their purchase of 
+Gluster in 2011.
```

Comment:
Gluster is essentially a community project and hence I don't think the ownership of Red Hat is to be pointed out in the documentation. Moreover, since this is a technical write-up let us focus more on what it does rather that on its history. :-)
@sac Thoughts?